scripts/spell-check.cmd

```diff
@@ -0,0 +1,20 @@
+@ECHO OFF
+
+pip --version || pip3 --version
+IF %ERRORLEVEL% NEQ 0 (
+    cls
+    ECHO Please install Python from https://www.python.org/downloads/ and Pip following https://pip.pypa.io/en/stable/installation/
+    EXIT /B 
+)
+
+cls
+codespell --version
+IF %ERRORLEVEL% NEQ 0 (
+    ECHO Installing codespell
+    pip install codespell || pip3 install codespell
+) else (
+    ECHO Codespell already installed
+)
+cls
+codespell -I scripts/resources/spell-check-ignore-list.txt --skip="*.svg,*.dxf" content
+ECHO Spell-check finished
```

Comment:
```suggestion
```
I'd keep the output the same as with the Unix version. VSCode should indicate when the task is done, so imo no extra message needed.

Reply:
Oops, didnt look at the unix version echo :S
Removing it